docs/src/utils/exampleKebabNameToFilename.ts

```diff
@@ -0,0 +1,15 @@
+import * as _ from 'lodash/fp'
+
+/**
+ * Converts kebab-cased-example-name back into the original filename.
+ * @param {string} exampleKebabName
+ */
+const exampleKebabNameToFilename = exampleKebabName => {
+  // button-example           => ButtonExample.tsx
+  // button-example-shorthand => ButtonExample.shorthand.tsx
```

Comment:
ideally we would like it to be expressed in tests

Reply:
Yes, the entire doc experience is untested.  Only `src` has tests currently.

There should be a larger effort to abstract the docs into a separate project that works for others and is also tested.